src/components/studies/participants/ParticipantManager.tsx

```diff
@@ -206,34 +208,111 @@ const ParticipantManager: FunctionComponent<ParticipantManagerProps> = () => {
 
   const {data: scheduleEvents = [], error: eventError} = useEvents(studyId)
 
-  const {
-    data,
-    status,
-    error,
-    run,
-    setData: setParticipantData,
-  } = useAsync<ParticipantData>({
-    status: 'PENDING',
-    data: null,
-  })
-
-  React.useEffect(() => {
-    if (!study?.identifier) {
-      return
-    }
-    const fn = async () => {
-      const result: ParticipantData = await run(
-        ParticipantUtility.getParticipants(
-          study.identifier,
+  // const {
+  //   data,
+  //   status,
+  //   error,
+  //   run,
+  //   setData: setParticipantData,
+  // } = useAsync<ParticipantData>({
+  //   status: 'PENDING',
+  //   data: null,
+  // })
+  // console.log("DATA",data?.items)
+
+  const {data: data, error : participantError, status: status} = useParticipants(study?.identifier,currentPage,pageSize,tab)
+  
+  const {    
+    mutate,
+    isSuccess,
+    isError,
+    isLoading: isParticipantUpdating,
+  } = useUpdateParticipantInList()
+  
+  const [curStatus, setCurStatus] = React.useState<RequestStatus>('PENDING')
```

Comment:
I don't think we need  a state variable here. Can we just use the `status` from the `useParticipants`?